packages/core/docs/recipes/redis.md

```diff
@@ -0,0 +1,101 @@
+# Use of redis{Cluster|Sentinel} plugins
+Please read [redisCluster](../reference/redis/cluster.md) and [redisSentinel](../reference/redis/sentinel.md) plugins reference first.
+
+## Recipies:
+* [Redis Cluster](#connect-to-redis-cluster)
+* [Redis Sentinel](#connect-to-redis-sentinel)
+* [HTTP action with Redis command](#create-action-using-redis-family-plugins)
+* [Perform additional operations on Successfull connection](#perform-additional-operations)
+## Connect to Redis Cluster
+### 1. Enable plugin
+Enable `RedisCluster` plugin in your service configuration by adding it into `config.plugins` list:
+
+```js
+// configs/plugins.js
+exports.plugins = [
+  'validator',
+  'redisCluster'
+];
+```
+
+### 2. Plugin configuration
+Connect to Redis Cluster with nodes running on the same server but with different port:
+```js
+// configs/redis.js
+exports.redis = {
+  options: {
+    keyPrefix: '{myPrefix}',
+    dropBufferSupport: false,
+  },
+  luaScripts: '/path/to/my/cool-lua-scripts/',
```

Comment:
Maybe we will go with the minimum possible configuration? It could help the reader to focus on what does really matter here. We could omit `luaScripts` and `dropBufferSupport` here.